File: cpp/2235.cpp
#include <iostream>
#include <cmath>

using namespace std;

int main()
{
    int a, b, c, count = 0;
    cin >> a >> b >> c;

    while (true) {
        if (a + b == c || a == b || abs(a - b) == c) {
            cout << "S\n";
            break;
        } else if (count == 3) {
            cout << "N\n";
            break;
        }

        a = a + c;
        c = a - c;
        a = a - c;
        a = a + b;
        b = a - b;
        a = a - b;
        count++;
    }

    return 0;
}

File: py/2235.py
credits = [int(i) for i in input().split()]

count = 0
while True:
    if credits[0] + credits[1] == credits[2] or credits[0] == credits[1] or abs(credits[0] - credits[1]) == credits[2]:
        print('S')
        break
    
    elif count == 3:
        print('N')
        break
    
    count += 1
    credits = credits[1:] + credits[:1]